contracts/Market.sol

```diff
@@ -292,59 +287,84 @@ contract Market is IMarket, Ownable, ERC721 {
 		// add to the total potential payout for this proposition
 		_potentialPayout[propositionId] += payout;
 
+		uint64 index = _getCount();
```

Comment:
the `Placed` event previously emitted `_getCount() - 1`, just wondering why we now want to emit `_getCount()`

Reply:
I believe the index was out by 1